public/v2/apps/limesurvey.json

```diff
@@ -0,0 +1,69 @@
+{
+  "captainVersion": "2",
+  "displayName": "Limesurvey",
+  "description": "Limesurvey is an open source online survey tool.",
+  "dockerCompose": {
+    "version": "2",
+    "services": {
+      "$$cap_appname-db": {
+        "documentation": "Taken from https://hub.docker.com/_/mariadb",
+        "image": "mariadb:$$cap_mariadb_version",
+        "notExposeAsWebApp": "true",
+        "volumes": [
+          "$$cap_appname-db-data:/var/lib/mysql"
+        ],
+        "restart": "always",
+        "environment": {
+          "MYSQL_ROOT_PASSWORD": "$$cap_db_pass",
+          "MYSQL_DATABASE": "$$cap_db_name",
+          "MYSQL_USER": "$$cap_db_user",
+          "MYSQL_PASSWORD": "$$cap_db_pass"
+        }
+      },
+      "$$cap_appname": {
+        "documentation": "Taken from https://registry.hub.docker.com/r/crramirez/limesurvey",
+        "depends_on": [
+         "$$cap_appname-db"
+        ],
+        "image": "crramirez/limesurvey:$$cap_limesurvey_version",
+        "volumes": [
+          "$$cap_appname-data:/app/upload"
+        ],
+        "restart": "always"
+      }
+    }
+  },
+  "instructions": {
+    "start": "Limesurvey is an open source online survey tool.\nThe process will deploy a database and the App.\nThe process will take about a minute for the process to finish.",
+    "end": "Limesurvey is deployed and available as $$cap_appname.\n\nPlease use srv-captain--$$cap_appname-db as database host on your online configuration.\n$$cap_db_user as user name\n$$cap_db_pass as password\n$$cap_db_name as database name"
+  },
+    "variables": [
+    {
+      "id": "$$cap_limesurvey_version",
+      "label": "Limesurvey Version",
+      "defaultValue": "3",
+      "description": "https://registry.hub.docker.com/r/crramirez/limesurvey/tags",
+      "validRegex": "/^([^\\s^\\/])+$/"
+    },
+    {
+      "id": "$$cap_mariadb_version",
+      "label": "MariaDB (database) version",
+      "defaultValue": "10.5.3",
+      "description": "Check out their Docker page for the valid tags https://hub.docker.com/_/mariadb?tab=tags"
+    },{
+      "id": "$$cap_db_pass",
+      "label": "database password",
+      "description": "Password for the database user and root using mysql."
+    },{
+      "id": "$$cap_db_name",
+      "label": "database name",
+      "defaultValue": "limesurvey",
+      "description": "name for the database."
+    },{
+      "id": "$$cap_db_user",
+      "label": "database user",
+      "defaultValue": "limesurvey",
+      "description": "Username for the database using mysql."
```

Comment:
valid regex is missing from multiple params. Please add them.

Reply:
done